src/sub.fs

```diff
@@ -0,0 +1,88 @@
+namespace Elmish
+
+open System
+
+/// SubId - Subscription ID, alias for string
+type SubId = string
+
+/// Subscription - Generates new messages when running
+type Sub<'msg> = { Start: SubId -> Dispatch<'msg> -> IDisposable }
+
+module Sub =
+
+    module One =
+
+        /// When emitting the message, map to another type
+        let map (f: 'a -> 'msg) (sub: Sub<'a>) : Sub<'msg> =
```

Comment:
This is why I was wondering if SubId should be in Sub - here we don't have the way to add prefix.

Reply:
I think you're right about the ID. Especially with what I'm about to post.

Reply:
I was trying to get rid of the Sub/Subs distinction like Cmd, but it feels like I'll have to bring it back.

Reply:
The one-letter difference between Sub and Subs is too subtle I think. Especially if the same named fn `map` is going to exist for both. I could just remove the individual map. Cmd takes this approach, there was no map fn for an individual item.

Reply:
Or do you like the `Sub.One.map` API?

Reply:
I'd be ok removing `One` module entirely.